guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -28,10 +28,16 @@ View compliance reports.
 | Discovery Reader | View hosts and discovery rules.
 | Edit hosts | View, create, edit, destroy, and build hosts.
 | Edit partition tables | View, create, edit and destroy partition tables.
+ifdef::orcharhino[]
+| ForemanRhCloud | View hosts inventory, generate and download reports, upload reports to {RHCloud}.
+This role depends on the Red Hat Insights plugin.
+endif::[]
 | Manager | View and edit global settings.
 | Organization admin | An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
+ifdef::satellite[]
 | Red{nbsp}Hat Access Logs | View the log viewer and the logs.
```

Comment:
I had to look it up, but this comes from https://github.com/redhataccess/foreman-plugin. We indeed do not ship this in upstream so the ifdef is correct.